components/core/Landing.js

```diff
@@ -425,6 +581,82 @@ const FileSupportCardItem = styled.div`
 
 const Landing = ({ setOpen, user, setUser }) => {
     
+  const [data, setData] = useState([]); // shapes
+  const [loading, setLoading] = useState(true); // shapes loading
+
+  useEffect(async () => {
```

Comment:
@williamzhu17 Everything related to trending shapes move to a separate component. In landing, just call the component.

Reply:
@atapas All code relating to trending shapes should now be a separate component named TrendingShapes.

Reply:
@williamzhu17 The changes are lookig good. I have made a few more refactoring and committed. Please check once. I shall merge it  soom. Resolving this comment.